submissions/Semka094/tiny-js-world/index.js

```diff
@@ -0,0 +1,54 @@
+import {print} from './js/lib.js';
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Semka094/a-tiny-JS-world
+   Web app: https://semka094.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+
+const cat = {
+  name: 'Knopa',
+  gender: 'female',
+  species: 'animal',
+  legs: 4,
+  hands: undefined,
+  saying: "miay!"
+};
+
+const dog = {
+  name: 'Bubluk',
+  gender: 'male',
+  species: 'animal',
+  hands: undefined,
+  legs: 4,
+  saying: "woof-woof!"
+};
+
+const woman = {
+  name: 'Olena',
+  saying: 'Hello there',
+  gender: 'female',
+  species: 'human',
+  hands: 2,
+  legs: 2,
+};
+
+const man = {
+  name: 'Kostia',
+  saying: 'Hi, folks!',
+  gender: 'male',
+  species: 'human',
+  hands: 2,
+  legs: 2,
+};
+
+const inhabitants = [cat, dog, woman, man];
+const inhabitantKeys = ['species', 'name', 'gender', 'legs', 'hands', 'saying'];
+
+inhabitants.forEach((inhabitant) => {
+  print(
+    inhabitantKeys.map((key) => inhabitant[key] ?? 'N/A').join('; ')
```

Comment:
> Variable names should not be too general
Also see [here](https://kottans.org/documentation/docs/doc/code-review/#code-style)

Inhabitants are characterized by properties or features. `key` here is obviously a generic technical term.
There is a reason inhabitants' container is named `inhabitants` rather than `objects` and the dog is named `dog` rather than `object5`.

Reply:
It is fixed. Please, review.